### PR TITLE
[API] Create Swimlane

### PR DESCRIPTION
**Pull request by yamila-moreno**
This PR creates the endpoints to CRUD swimlanes. **Important** this US is specifically for _creation_, not for update or deletion (they'll come after this).

- [x] check the code
- [x] run the tests, at least the new tests \o/
- [x] you've `jq` installed
- [x] delete the database an launch a new sample_data
```sh
$ python manage.py migrate
$ python manage.py loaddata initial_user
$ python manage.py loaddata initial_project_templates
$ python manage.py sample_data
```
- [x] runserver
- [x] ensure you're using a user who is admin in the project
- [ ] execute this script (understand it first!)
```sh
TOKEN=$(
        curl -X POST \
            -H "Content-Type: application/json" \
            -d '{"type": "normal", "username": "user1", "password": "123123"}' \
            http://localhost:8000/api/v1/auth | jq -r '.auth_token'
        )

curl -X POST \
    -H "Content-Type: application/json" \
    -H "Authorization: Bearer $TOKEN" \
    -d '{"name": "Swimlane 1", "project": 1}' \
    http://localhost:8000/api/v1/swimlanes

curl -X GET \
    -H "Content-Type: application/json" \
    -H "Authorization: Bearer $TOKEN" \
    http://localhost:8000/api/v1/swimlanes?project=1
```
- [x] you can test as well trying this without being _admin_ (should get: error PermissionDenied)
- [x] you can test as well with a user _admin_ in a blocked project (should get: error Blocked)
- [x] move task 713

### CODE REVIEW

File: taiga/projects/api.py [hunk 1]
```diff
@@ -642,6 +642,35 @@ def create(self, request, *args, **kwargs):
...
+
+    def post_save(self, object, created=False):
+        super().post_save(object, created=created)
+
+        if not created:
+            return
+
+        # If it's a creation and it's the first and only swimlane,
+        # then assign all Userstories to this new swimlane
+        total_swimlanes = self.model.objects.filter(project=object.project).count()
```
Comment (bameda) [inline taiga/projects/api.py:667]: maybe 
```
total_swimlanes = object.project.swmlanes.count()
```

File: taiga/projects/api.py [hunk 2]
```diff
@@ -642,6 +642,35 @@ def create(self, request, *args, **kwargs):
...
+        if total_swimlanes == 1:
+            us_model = apps.get_model("userstories", "UserStory")
+            uss = us_model.objects.filter(project=object.project)
```
Comment (bameda) [inline taiga/projects/api.py:670]: Maybe
```
object.project.user_stories.all()
```

File: taiga/projects/api.py (hunk 1 above)
Comment (yamila-moreno) [reply taiga/projects/api.py:667]: ostias, sí, mucho mejor, ahora voy

File: taiga/projects/api.py (hunk 2 above)
Comment (yamila-moreno) [reply taiga/projects/api.py:670]: también! mejor. Estaba haciendo _mutatis mutandis_ pero queda así mucho mejor. voy!

**Pull request merged**